docs/products/add-data-to-grid-datastores/README.md

```diff
@@ -17,6 +17,37 @@ Datastores can be attached to Runs or Sessions and preserve the file format and
 We don't charge for data storage!
 :::
 
+## Datastores with Private S3 Buckets
+
+We only recommend using private S3 buckets for BYOC users. This can be accomplished with the following steps:
```

Comment:
This isn't a recomendation. It is the only thing we are willing to support / guarantee right now.